src/App.jsx

```diff
@@ -14,70 +15,79 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const activeGoodClass = 'has-background-success-light';
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const onAddHandler = good => setSelectedGood(good);
+  const onClearHandler = () => setSelectedGood('');
```

Comment:
handleGoodAdd, addGood, something like this, but not with 'handler' in the end of name
clearSelectedGood, clearGood, handleGoodClear